packages/bbui/src/Drawer/Drawer.svelte

```diff
@@ -4,27 +4,34 @@
   import Body from "../Typography/Body.svelte"
   import Heading from "../Typography/Heading.svelte"
   import { setContext } from "svelte"
+  import { createEventDispatcher } from "svelte"
+  import { generate } from "shortid"
 
   export let title
   export let fillWidth
   export let left = "314px"
   export let width = "calc(100% - 626px)"
   export let headless = false
 
+  const dispatch = createEventDispatcher()
+
   let visible = false
+  let drawerId = generate()
 
   export function show() {
     if (visible) {
       return
     }
     visible = true
+    dispatch("drawerShow", drawerId)
```

Comment:
Just wondering if you ever use this drawer ID? I can't see it being used, and you have to pass `on:drawerShow` handler to an instance of a drawer anyway, so you'll probably know which drawer this is.

Reply:
Its picked up and stored in the `EditFieldPopover` component to help discern how many unique drawers have been opened.